godel_simple_gui/src/options/scan_tool_configuration.cpp

```diff
@@ -0,0 +1,55 @@
+#include "godel_simple_gui/options/scan_tool_configuration.h"
+#include "ui_scan_tool_configuration.h"
+
+godel_simple_gui::ScanPlanConfigWidget::ScanPlanConfigWidget(godel_msgs::ScanPlanParameters params)
+  : params_(params)
+{
+  ui_ = new Ui::ScanToolConfigWindow();
+  ui_->setupUi(this);
+
+  connect(ui_->PushButtonAccept, SIGNAL(clicked()), this, SLOT(accept_changes_handler()));
+  connect(ui_->PushButtonCancel, SIGNAL(clicked()), this, SLOT(cancel_changes_handler()));
+  connect(ui_->PushButtonSave, SIGNAL(clicked()), this, SLOT(save_changes_handler()));
+
+  ui_->ComboBoxQualityMetric->addItems(quality_metric_list);
+}
+
+QStringList godel_simple_gui::ScanPlanConfigWidget::quality_metric_list = QStringList() << "rms" << "ra";
```

Comment:
If we're going to switch from string based identification of scan scoring methods, then the associated methods here need to change as well.